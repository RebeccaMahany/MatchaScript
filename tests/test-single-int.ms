1;
print(1);
print("we're all here awaiting for our silver invitation");
print("to the big band in the sky");

