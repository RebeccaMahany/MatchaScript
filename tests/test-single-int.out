1
we're all here awaiting for our silver invitation
to the big band in the sky
